include/mata/nfa.hh

```diff
@@ -525,19 +597,77 @@ public:
         m_num_of_requested_states = 0;
     }
 
+    // this is exact equality of automata, including state numbering (so even stronger than isomorphism)
+    // essentially only useful for testing purposes
+    bool is_equal(const Nfa & aut) {
```

Comment:
Then, in my opinion, it should be names differently. No one will remember that `Mata::Nfa::are_equivalent` and `Mata::Nfa::Nfa::is_equal` does something different.

Reply:
Fixed.